lab-davide/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-davide",
+  "version": "1.0.0",
+  "description": "",
+  "main": "hello.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Instead of "echo \"Error: no test specified\" && exit 1" use "mocha", what this does is specify what the command npm test will perform, which in your case here, you'd want to be to run mocha. So line 10 should look like this:
"test": "mocha"